### PR TITLE
Feat/reset button

### PR DESCRIPTION
**Pull request by joaoqueiroz95**
image

### CODE REVIEW

**Comment by joaoqueiroz95** [comment]
[Reset ticket types](https://trello.com/c/R81xaU2C/33-reset-ticket-types)

File: src/hooks/useBookingStore.tsx [hunk 1]
```diff
@@ -50,7 +50,13 @@ export const useBookingStore = create<BookingStore>()((set, get) => ({
       .getGroupTickets()
       .flatMap(({ tickets }) => tickets);
   },
-  resetCounters: () => set(() => ({ groupTickets: [], tickets: [], itemCounters: {} })),
+  resetCounters: (group?: TicketTypeGroup["name"]) => {
+    if (!group) {
+      return set(() => ({ itemCounters: {} }));
+    }
+
+    set(state => ({ ...state, itemCounters: _.omit(state.itemCounters, group) }));
```
[the remark this answers is not in the document]
Comment (joaoqueiroz95) [reply src/hooks/useBookingStore.tsx]: Done

**Review by mathvaillant — APPROVED** [review]

**Comment by mathvaillant** [comment]
LGTM!

File: src/screens/TicketTypes.tsx [hunk 2]
```diff
@@ -43,47 +53,59 @@ export function TicketTypesScreen() {
...
+                  onPress={() =>
+                    navigate(NavigationScreens.TICKET_SELECTION, { ticketType: name })
+                  }>
+                  {name}
+                </TextButton>
+              )}
+              keyExtractor={({ key }) => String(key)}
+              ItemSeparatorComponent={separator}
+            />
+          )}
```
[the remark this answers is not in the document]
Comment (joaoqueiroz95) [reply src/screens/TicketTypes.tsx]: Just fixed

**Review by mathvaillant — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]
Looking Good

**Pull request merged**